covalent_dispatcher/_service/app.py

```diff
@@ -77,3 +79,9 @@ def cancel() -> Response:
     dispatcher.cancel_running_dispatch(dispatch_id)
 
     return jsonify(f"Dispatch {dispatch_id} cancelled.")
+
+
+@bp.route("/db-path", methods=["GET"])

```

Comment:
route endpoint `/db/path` to match the REST resource style better?

Reply:
We can come back to this as an enhancement later onwards!